libs/LibEPGPChat-1.0.lua

```diff
@@ -4,6 +4,10 @@ local MINOR_VERSION = 10000
 local lib, oldMinor = LibStub:NewLibrary(MAJOR_VERSION, MINOR_VERSION)
 if not lib then return end
 
+function lib:Interp(s, tab)
+  return (s:gsub('($%b{})', function(w) return tab[w:sub(3, -2)] or w end))
+end
```

Comment:
Please put this function between `Announce` and `Whisper` according to alphabetical order.

Reply:
done!